website/docs/r/iam_service_specific_credential.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "IAM"
+layout: "aws"
+page_title: "AWS: aws_iam_service_specific_credential"
+description: |-
+  Provides an IAM Service Specific Credential.
+---
+
+# Resource: aws_iam_service_specific_credential
+
+Provides an IAM Service Specific Credential.
+
+## Example Usage
+
+```hcl
+resource "aws_iam_user" "example" {
+  name = "example"
+}
+
+resource "aws_iam_service_specific_credential" "example" {
+  service_name = "codecommit.amazonaws.com"
+  user_name    = aws_iam_user.example.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `service_name` - (Required) The name of the AWS service that is to be associated with the credentials. The service you specify here is the only service that can be accessed using these credentials.
+* `user_name` - (Required) The name of the IAM user that is to be associated with the credentials. The new service-specific credentials have the same permissions as the associated user except that they can be used only to access the specified service.
+* `status` - (Optional) The status to be assigned to the service-specific credential. Valid values are `Active` and `Inactive`. Default value is `Active`.
```

Comment:
Should we say that the initial status will always be `Active`, and then Terraform will switch the value to `Inactive` or not, depending on the value that the user provides? Because the AWS API does not allow the `status` field to be set during creation.

Reply:
As the key is not given until the full create cycle is complete i dont think its necessary.